src/cloud/components/Editor/index.tsx

```diff
@@ -440,7 +461,18 @@ const Editor = ({
         500,
         { trailing: true }
       )
-    )
+    })
+
+    // editor.on('endCompletion', (cm) => {
```

Comment:
Why does this exist?

Reply:
Two approaches of doing suggestions (auto-open, auto close)
First commit is first way, second second

But we should discuss how to handle it, do we have a way of knowing that suggestion has been completed/selected, and afterwards how to save it to a state of the component?

Because not sure if there is ID of the suggestion/completion - since user can have multiple code blocks and multiple suggestions in one document, so question arises for when to give/open suggestions again?